k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageTopView.java

```diff
@@ -126,6 +126,10 @@ public void setOnToggleFlagClickListener(OnClickListener listener) {
         mHeaderContainer.setOnFlagListener(listener);
     }
 
+    public void setOnToggleFlagLongClickListener(OnLongClickListener listener){
```

Comment:
This change seems unrelated to copying email addresses to the clipboard. Please limit the scope of pull requests as much as possible.